### ISSUE TITLE
"InlineContainer

### ISSUE THREAD

**Issue by dgerding** [CXuesong/MwParserFromScratch#10]
I'm getting an "inline container missing" error when I try and build the solution.

Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0246	The type or namespace name 'InlineContainer' could not be found (are you missing a using directive or an assembly reference?)	MwParserFromScratch(net45), MwParserFromScratch(netstandard1.1)	D:\Repos\MwParserFromScratch-master\MwParserFromScratch\ParserCore.Basic.cs	237	Active

Let me know if the file is there.  I'll go back and look  :)

Dave G

**Comment by CXuesong** [comment]
Oops, I've forgot to commit a file.

I tried write table parsing logic months ago, but before I can finish it, I found something else interesting to do ;-) Perhaps later I will put my hands on it again. So now we have `Table` node, but there is current no parsing logic for it.

Btw, the code of latest stable release v0.2.0 can be found [here](https://github.com/CXuesong/MwParserFromScratch/tree/0.2.0).
